inference-engine/include/ie_api.h

```diff
@@ -90,10 +90,7 @@
 
 #ifndef ENABLE_UNICODE_PATH_SUPPORT
 # ifdef _WIN32
-#  ifdef __INTEL_COMPILER
-#   define ENABLE_UNICODE_PATH_SUPPORT
-#  endif
-#  if defined _MSC_VER && defined _MSVC_LANG && _MSVC_LANG < 201703L
+#  if defined __INTEL_COMPILER || defined _MSC_VER
```

Comment:
Am I right understand that these changes allow to expand a set of supported MSVC compilers? or anything else?

Reply:
It allows to support unicode for MSVC + C++17, currently it's disabled because unicode utils are deprecated in c++17.

I am going to continue with patch after https://github.com/openvinotoolkit/openvino/pull/1556 I need to move implementation to cpp files as well not to team Windows.h in public headers